features/__snapshots__/validate_image.snap

```diff
@@ -1276,10 +1276,7 @@ Error: success criteria not met
 ---
 
 [happy day with missing git config:stderr - 1]
-Error: 1 error occurred:
```

Comment:
What does it look like if there is more than one error? Are they joined by `\n`?

Reply:
Yeah, I missed some failing tests that had more than one error added in [`2bd4399` (#2008)](https://github.com/enterprise-contract/ec-cli/pull/2008/commits/2bd43998bac0891cd6dbad0a44876276f0f67050). For example this: [`2bd4399` (#2008)](https://github.com/enterprise-contract/ec-cli/pull/2008/commits/2bd43998bac0891cd6dbad0a44876276f0f67050#diff-e24f93dbfa8e364e10dfe82831576305eb4a9e9c83eb621e06c7fbf489de695fR2042-R2044) snapshot has three errors joined